### ISSUE TITLE
Installation issue - 502 bad gateway

### ISSUE THREAD

**Issue by jengajenga** [mediacms-io/mediacms#448]
**Describe the issue**
Server installation produces 502 bad gateway error after installation completion

**To Reproduce**
Following installation instructions
1. mkdir /home/mediacms.io && cd /home/mediacms.io/
2. git clone https://github.com/mediacms-io/mediacms
3. cd /home/mediacms.io/mediacms/ && bash ./install.sh

At the end of the installation, this comes up

_"Resolving zebulon.bok.net (zebulon.bok.net)... failed: Temporary failure in name resolution.
wget: unable to resolve host address ‘zebulon.bok.net’
unzip:  cannot find or open Bento4-SDK-1-6-0-637.x86_64-unknown-linux.zip, Bento4-SDK-1-6-0-637.x86_64-unknown-linux.zip.zip or Bento4-SDK-1-6-0-637.x86_64-unknown-linux.zip.ZIP."_

And then I do the update
1. cd /home/mediacms.io/mediacms # enter mediacms directory
2. source  /home/mediacms.io/bin/activate # use virtualenv
3. git pull # update code
4. python manage.py migrate # run Django migrations
5. sudo systemctl restart mediacms celery_long celery_short # restart services

At the end of the update, this comes up

_"Traceback (most recent call last):
  File "manage.py", line 13, in <module>
    execute_from_command_line(sys.argv)
  File "/home/mediacms.io/lib/python3.8/site-packages/django/core/management/__init__.py", line 446, in execute_from_command_line
    utility.execute()
  File "/home/mediacms.io/lib/python3.8/site-packages/django/core/management/__init__.py", line 440, in execute
    self.fetch_command(subcommand).run_from_argv(self.argv)
  File "/home/mediacms.io/lib/python3.8/site-packages/django/core/management/base.py", line 414, in run_from_argv
    self.execute(*args, **cmd_options)
  File "/home/mediacms.io/lib/python3.8/site-packages/django/core/management/base.py", line 460, in execute
    output = self.handle(*args, **options)
  File "/home/mediacms.io/lib/python3.8/site-packages/django/core/management/base.py", line 98, in wrapped
    res = handle_func(*args, **kwargs)
  File "/home/mediacms.io/lib/python3.8/site-packages/django/core/management/commands/migrate.py", line 91, in handle
    self.check(databases=[database])
  File "/home/mediacms.io/lib/python3.8/site-packages/django/core/management/base.py", line 487, in check
    all_issues = checks.run_checks(
  File "/home/mediacms.io/lib/python3.8/site-packages/django/core/checks/registry.py", line 88, in run_checks
    new_errors = check(app_configs=app_configs, databases=databases)
  File "/home/mediacms.io/lib/python3.8/site-packages/django/core/checks/urls.py", line 14, in check_url_config
    return check_resolver(resolver)
  File "/home/mediacms.io/lib/python3.8/site-packages/django/core/checks/urls.py", line 24, in check_resolver
    return check_method()
  File "/home/mediacms.io/lib/python3.8/site-packages/django/urls/resolvers.py", line 480, in check
    for pattern in self.url_patterns:
  File "/home/mediacms.io/lib/python3.8/site-packages/django/utils/functional.py", line 49, in __get__
    res = instance.__dict__[self.name] = self.func(instance)
  File "/home/mediacms.io/lib/python3.8/site-packages/django/urls/resolvers.py", line 696, in url_patterns
    patterns = getattr(self.urlconf_module, "urlpatterns", self.urlconf_module)
  File "/home/mediacms.io/lib/python3.8/site-packages/django/utils/functional.py", line 49, in __get__
    res = instance.__dict__[self.name] = self.func(instance)
  File "/home/mediacms.io/lib/python3.8/site-packages/django/urls/resolvers.py", line 689, in urlconf_module
    return import_module(self.urlconf_name)
  File "/usr/lib/python3.8/importlib/__init__.py", line 127, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1014, in _gcd_import
  File "<frozen importlib._bootstrap>", line 991, in _find_and_load
  File "<frozen importlib._bootstrap>", line 975, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 671, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 848, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/mediacms.io/mediacms/cms/urls.py", line 2, in <module>
    from django.conf.urls import include, re_path
ImportError: cannot import name 're_path' from 'django.conf.urls' (/home/mediacms.io/lib/python3.8/site-packages/django/conf/urls/__init__.py)"_

**Expected behaviour**
I expected to see the user interface when i visit the domain name under which I install. Instead, I get a bad gateway 502/nginx.

**Screenshots**
![bad gateway](https://user-images.githubusercontent.com/65451390/167186806-225eadc6-3334-43b3-9bb8-9129878e936e.jpeg)

**Environment (please complete the following information):**
 - OS: UBUNTU 20.04
 - Installation method: SINGLE SERVER INSTALL
 - Browser: Edge, Firefox

**Additional context**
N/A

Any help would be highly appreciated.

Thank you.

**Comment by Donniewr** [comment]
Same issue here, Linode server.

**Comment by jengajenga** [comment]
Anyone who can assist me with this issue? @mgogoulos

**Comment by mgogoulos** [comment]
Hey, this seems serious, will try to reproduce it and apply a fix asap

**Comment by mgogoulos** [comment]
There were 1-2 issues with the existing versions, I've updated them and made sure installation completes. You might want to try out with the latest version (eg through `git pull`) or even try the docker way to install. 

The DNS issues seems to be unrelated. probably a temp network issue

**Comment by methosomega** [comment]
I had this issue on a docker installation had been running for a month. only occurs trying to login or access admin panel. fed up with it i left it 8 months later kids wanted it back so i bought a separate pc just to run their stuff.. fresh install and first thing I'm faced with is the same friggen error 502 just navigating to the interface for the first time.  any suggestions? I don't see anything in this thread that actually fixes or identifies the issue.

**Comment by mgogoulos** [comment]
There's a new major release and the server installation wasn't tested, so chances are that something is broken. Most probably something related with dependencies. 
It's hard to maintain the server installation version, along with the Docker one, I've been thinking of removing the server one...

**Comment by methosomega** [comment]
aww, I get that though.. probably easier to maintain dependencies and configurations via docker anyways. I just got a system specifically for it and didn't want to deal with docker overtop of the system.. I set this up as a way for my kids to upload videos and pretend to be youtubers between the family, Its only open to family networks.. keeps their videos off youtube until they're old enough to use good judgement of what they put on youtube. It was such a easy install and worked so well they had soo much fun with their cousins making videos. lol
